dpdd.tex

```diff
@@ -1222,20 +1226,37 @@ \subsubsection{Calibration Data}
+    The six per-band coadds will be kept indefinitely and made available to the users.\dmreq{0334} \emph{Their primary purpose (after data release production is complete) is to enable the end-users to apply alternative object characterization algorithms, perform studies of diffuse structures, and for visualization}.
 
-    \item One (ugrizyM) set of PSF-matched coadds.\dmreq{0335} These will be used to measure colors and shapes of objects at ``standard'' seeing. \textbf{\emph{We do not plan to keep and make these coadds available}}. We will retain and provide sufficient metadata for users to re-create them using Science Platform  or other resources.
+    \item A set of \emph{template coadds}.  These will be used for image differencing in at least that data release, and when possible will be used in the following season's Prompt Processing.  They will be retained and made available at least during the period in which they are used by Prompt Processing.  See also \S~\ref{sec:templates}.
 \end{itemize}
 
+DM will also provide software and configuration that will allow users to create additional coadds using the Science Platform or other resources. \dmreq{0311}\dmreq{0298}
+These will include at least PSF-matched coadds, multi-band detection coadds, yearly or other short-period coadds, and direct coadds built from images with only the visits with the best seeing. \dmreq{0330}\dmreq{0281}\dmreq{0335}\dmreq{0330}.
+The exact configurations for all coadds actually used in data release processing will be made available, even if the images themselves are not retained.
+
+% Changes on DM-23248 explicitly violates DMS-REQ-0334 (which coadds are
+% retained), removing "Best Seeing Coadds", because DRP isn't actually planning
+% to build or use them internally.  We could produce and serve them anyway to
+% meet the requiement, but I suspect there are much more scientifically
+% productive uses (e.g. postage-stamp coadds) for that part of the storage
+% budget, and I think it's irresponsible to promise to devote resources to
+% something we're not sure will be useful.
+%
+% Ideally, we would also update the text of DMS-REQ-{0281,0330,0334} to state
+% that DM "will deliver software to produce" these coadds rather than actually
+% producing them.  At least right now, DM has no need to produce them for its
+% internal use, and it's silly to promise to produce something that we are also
+% promising to throw away.
+
```

Comment:
I would deprecate (remove) DMS-REQ-{0281,0330} and update DMS-REQ-0334 and add a new requirement on DM to provide the software to produce various flavours of coadds.

Reply:
I also propose to deprecate(remove) DMS-REQ-0335 given that we will not generate nor persist psf-matched coadds. If users want to generate psf-matched coadds, this becomes another flavour of coadd that can be generated using software and configurations that we  provide.